rosdep/base.yaml

```diff
@@ -5647,6 +5647,13 @@ tmux:
   fedora: [tmux]
   gentoo: [app-misc/tmux]
   ubuntu: [tmux]
+trang:
+  debian: [trang]
+  fedora: [trang]
+  gentoo: [trang]
```

Comment:
This is coming from total Gentoo ignorance, but looking at other Gentoo rules, shouldn't this be `app-text/trang`?

Reply:
You are right https://github.com/ros-infrastructure/rosdep/blob/50ee23e855b4b09c30b1a71b56bdaf98267e6323/src/rosdep2/platforms/gentoo.py#L30-L40